src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/GeneratedCodeContainer.cs

```diff
@@ -153,6 +153,10 @@ public void SetOutput(RazorCSharpDocument csharpDocument, DefaultDocumentSnapsho
                 {
                     results.Add(new SpanMapResult(document, linePositionSpan));
                 }
+                else
+                {
+                    results.Add(null);
+                }
```

Comment:
This matches up with some of my other changes in the feature branch. We should return a result for each span they give us.